cg/services/bcl_convert_metrics_service/parser.py

```diff
@@ -175,3 +167,55 @@ def has_undetermined_reads_in_lane(self, lane: int) -> bool:
             metrics=self.quality_metrics, sample_internal_id=UNDETERMINED, lane=lane
         )
         return bool(metrics)
+
+    @classmethod
+    def calculate_total_reads_for_metrics(cls, read_pair_count: int) -> int:
+        """Scale to read pair number up to single reads."""
+        return read_pair_count * SCALE_TO_READ_PAIRS
+
+    def get_aggregate_total_reads_for_metrics(self):
+        """Return the aggregate reads for the whole demux metrics excluding indexchecks."""
```

Comment:
I see no logic regarding index checks
```suggestion
        """Return the aggregate reads for the whole demux metrics."""
```